src/worker.py

```
@@ -40,7 +40,7 @@
 import re
 import traceback
 from types import SimpleNamespace
-from typing import Any, Dict
+from typing import Any, Dict, Optional
```

Ruff (0.15.11)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)